src/res/translation/translation_pl_PL.ts

```diff
@@ -3206,57 +3206,57 @@ Twój kanał został wyciszony i włączono „Wycisz mnie”. Napraw przyczynę
     <message>
         <location filename="../../serverdlg.cpp" line="125"/>
         <source>Recording has been switched off by the UI checkbox</source>
-        <translation type="unfinished"></translation>
+        <translation>Nagrywanie zostało wyłączone przez użytkownika</translation>
     </message>
     <message>
         <location filename="../../serverdlg.cpp" line="127"/>
         <source>Recording has been switched off, either by the UI checkbox or SIGUSR2 being received</source>
-        <translation type="unfinished"></translation>
+        <translation>Nagrywanie zostało wyłączone przez użytkownika lub otrzymano komunikat SIGUSR2</translation>
```

Comment:
@dtinth (although unrelated to this PR) we need to update this if your PR #1975 gets merged: Recording has been switched off, by the UI checkbox, SIGUSR2 command or via JSON-RPC API